lib/theming/src/index.ts

```diff
@@ -1,8 +1,9 @@
-import styled from '@emotion/styled';
+import emotionStyled, { CreateStyled } from '@emotion/styled';
 import dark from './themes/dark';
 import light from './themes/light';
+import { Theme } from './base';
 
-export { styled };
+export const styled = emotionStyled as CreateStyled<Theme>;
```

Comment:
@ndelangen am I right to type `styled` with `Theme` or should I had used `ThemeVars`?

Reply:
The components will receive the `Theme`